src/mixins/view.js

```diff
@@ -242,7 +238,13 @@ const ViewMixin = {
       this.triggerMethod(childViewTriggers[eventName], ...args);
     }
 
-    this.triggerMethod(childEventName, ...args);
+    const prefix = _.result(this, 'childViewEventPrefix');
+
+    if (prefix !== false) {
```

Comment:
can't this be `if (_.isString(prefix)) {`.
Seeing this makes me think that anything other than `false` is acceptable.